README: Add instructions on cloning with submodules (#622)
bluerobotics/cockpit: README.md

## Installing Cockpit
Cockpit is typically installed as an Electron application or BlueOS Extension, but for development purposes it is valuable to run it locally.

> First of all, you need to clone the repository and it's submodules. Don't forget to clone the submodules as they are needed for Cockpit to run:

Reviewer: ```suggestion
To start, clone the repository and its submodules. The submodules are required components for Cockpit to be able to run:
```

```
git clone --recurse-submodules git@github.com:bluerobotics/cockpit.git
```

> After that you should [install bun](https://bun.sh/docs/installation). We really recomend following their documented instructions, as it was already reported by some that installing `bun` from alternatives like Snap has not worked properly.

Reviewer: ```suggestion
Next, [install bun](https://bun.sh/docs/installation). Following their documented instructions is strongly recommended - we have received reports that installing `bun` from alternatives like Snap did not work properly.
```
